### PR TITLE
Feature/dev 9146

### PR DESCRIPTION
**Pull request by Atash3000**
## [Replace With Ticket Number]
Fixed spacing on message, filters and legend (when legend if top)
October sprint.

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

### CODE REVIEW

File: packages/core/styles/filters.scss [hunk 1]
```diff
@@ -5,6 +5,12 @@
     gap: 7px 15px;
     margin-top: 15px;
     margin-bottom: 15px;
+    label {
```
Comment (adamdoe) [inline packages/core/styles/filters.scss:8]: fix: can we try using bootstraps font classes here and remove the class?

`fw-bolder`, `fs-6`, `mb-[number]` maybe?

File: packages/core/styles/_global.scss [hunk 2]
```diff
@@ -183,9 +183,6 @@ select {
 section.introText {
   width: 100%;
   margin-bottom: 25px;
-  &.legend_visible_top {
```
Comment (adamdoe) [inline packages/core/styles/_global.scss:186]: good!

**Review by adamdoe — CHANGES_REQUESTED** [review]

File: packages/core/components/Filters.tsx [hunk 3]
```diff
@@ -478,16 +478,23 @@ const Filters = (props: FilterProps) => {
...
 
   if (visualizationConfig?.filters?.length === 0) return
-  const filterSectionClassList = [
-    `filters-section legend_${visualizationConfig?.legend?.hide ? 'hidden' : 'visible'}_${
-      visualizationConfig?.legend?.position || ''
-    }`,
-    type === 'map' ? general.headerColor : visualizationConfig?.visualizationType === 'Spark Line' ? null : theme
-  ]
+
+  const getCLasses = () => {
```
Comment (joshlacey) [inline packages/core/components/Filters.tsx:482]: getClasses

File: packages/core/components/Filters.tsx [hunk 4]
```diff
@@ -478,16 +478,23 @@ const Filters = (props: FilterProps) => {
...
+    const classList = [
+      'filters-section',
+      type === 'map' ? general.headerColor : visualizationConfig?.visualizationType === 'Spark Line' ? null : theme
```
Comment (joshlacey) [inline packages/core/components/Filters.tsx:485]: return empty string here  so that null doesn't get interpolated into the classname

**Review by joshlacey — CHANGES_REQUESTED** [review]

File: packages/core/components/Filters.tsx (hunk 3 above)
Comment (Atash3000) [reply packages/core/components/Filters.tsx]: Fixed!

**Review by mpallansch — APPROVED** [review]

**Pull request merged**